_build_dependencies.sh

```diff
@@ -1,2 +1,11 @@
 # This file has all the information on what versions of libraries are thrown into the code
+# For dreamchecker
 export SPACEMANDMM_TAG=suite-1.2
+# For NanoUI
+export NODE_VERSION=9
+# For the scripts in tools
+export PHP_VERSION=5.6
+# Byond Major
+export BYOND_MAJOR=513
+# Byond Minor
+export BYOND_MINOR=1505
```

Comment:
Hmm annoying to have to update these using PRs everytime. Out of the scope of the PR though

Reply:
Still easier than the old way which is finding the vars in the `.travis.yml` file. This is now actually properly laid out.